google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudLibraryRenderer.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.apis;
+
+import com.google.cloud.tools.libraries.json.CloudLibrary;
+import java.awt.Component;
+import javax.swing.JTable;
+import javax.swing.border.Border;
+import javax.swing.border.EmptyBorder;
+import javax.swing.table.DefaultTableCellRenderer;
+
+/**
+ * Custom {@link javax.swing.table.TableCellRenderer TableCellRenderer} for {@link CloudLibrary}
+ * objects.
+ */
+final class CloudLibraryRenderer extends DefaultTableCellRenderer {
+
+  private static final Border NO_FOCUS_BORDER = new EmptyBorder(5, 5, 5, 5);
+
+  @Override
+  public Component getTableCellRendererComponent(
+      JTable table, Object value, boolean isSelected, boolean hasFocus, int row, int column) {
+    Component component =
+        super.getTableCellRendererComponent(table, value, isSelected, hasFocus, row, column);
+    setBorder(NO_FOCUS_BORDER);
```

Comment:
Please correct me if I'm wrong, do we set the `NO_FOCUS_BORDER` only to remove "selected" state of the cell in the table? What do you think if we just pass `isSelected` and `hasFocus` as always `false` instead into `super.getTableCellRendererComponent`? It might be a bit easier to understand and we can directly return the result of `super` call.

Reply:
this was code brought over from a previous PR (I just gave it its own class since its now reused).

But that being said, I'll try your suggestion.

Reply:
Actually CloudLibraryRenderer is used in two contexts, one of which requires selection (the first step of the wizard). So I don't think your strategy will work.